src/MediaCodecCryptoInfoPattern.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
copyright must be on top, place pragma below it

Reply:
FYI, this differs from C, where ifdef-guards came first. That was because the preprocessor literally transcluded less text when placed above the header comment. When we moved to C++, however, #pragma keeps an index of files and skips the file if the path is the same. So there's no (neglibigle) speed difference to care about, and the header comment might as well come first.

Reply:
it is ok, just noticed that most of the time in kodi projects this order is followed,
it was just to keep everything consistent nothing mandatory